src/test/java/cloud/fogbow/accs/core/processors/ProcessorsBaseUnitTest.java

```diff
@@ -0,0 +1,30 @@
+package cloud.fogbow.accs.core.processors;
+
+import cloud.fogbow.accs.core.datastore.orderstorage.AuditableOrderStateChange;
+import cloud.fogbow.accs.core.models.Record;
+import cloud.fogbow.accs.core.models.orders.Order;
+
+import java.sql.Timestamp;
+
+public class ProcessorsBaseUnitTest {
```

Comment:
It may be more effective to use a BaseUtitTests class in the cloud.fogbow.accs.core package containing common settings between SyncProcessorTest and RecordServiceTest classes, rather than specific classes for each.

This class could even contain all of this necessary annotation set up, reducing the complexity of the test classes that use it.